src/main/java/io/sdkman/broker/download/Platform.java

```diff
@@ -8,6 +8,8 @@
 
 public enum Platform {
     FREE_BSD(newArrayList("FreeBSD")),
+    LINUX_ARM32(newArrayList("LinuxARM32")),
+    LINUX_ARM64(newArrayList("LinuxARM64")),
```

Comment:
Be aware that order of the enums do matter here due `startsWith()` check at https://github.com/sdkman/sdkman-broker/blob/8e6e61e202aed657b993a1583d3b02a8d764ae18/src/main/java/io/sdkman/broker/download/Platform.java#L28

Reply:
That's right, so as you have it should work, with more specialised entries first.